Source/WebKit/UIProcess/API/wpe/WPEWebView.cpp

```diff
@@ -283,6 +277,22 @@ View::View(struct wpe_view_backend* backend, WPEDisplay* display, const API::Pag
                     break;
                 }
             }
+            if (changedMask & WPE_TOPLEVEL_STATE_ACTIVE) {
+                OptionSet<WebCore::ActivityState> flagsToUpdate;
```

Comment:
This is quite hard to read, and already improves a lot IMHO, when placing WebCore::ActivityState::WindowIsActive in a local variable:

```
                OptionSet<WebCore::ActivityState> flagsToUpdate;
                constexpr auto flagToCheck { WebCore::ActivityState::WindowIsActive };

                if (state & WPE_TOPLEVEL_STATE_ACTIVE) {
                    if (!webView.m_viewStateFlags.contains(flagToCheck)) {
                        flagsToUpdate.add(flagToCheck);
                        webView.m_viewStateFlags.add(flagToCheck);
                    }
                } else {
                    if (webView.m_viewStateFlags.contains(flagToCheck)) {
                        flagsToUpdate.add(flagToCheck);
                        webView.m_viewStateFlags.remove(flagToCheck);
                    }
                }
```

Reply:
I'm fine, but I don't understand it to be honest, why is easier to read just using a local variable? is it because you think the lines are too long with the value directly?